viewing_party/party.py

```diff
@@ -1,23 +1,162 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    new_movie = {}
+    if title and genre and rating:
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
 
+        return new_movie
+
+def add_to_watched(user_data, movie):
+    '''
+    user data is a dictionary containing a key "watched" with value []
+    "watched" will be a list of dictionaries the user has watched
+    '''
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    '''
+    user data is dictionary containing value "watchlist with value []
+    '''
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, MOVIE_TITLE_1):

```

Comment:
The parameter name for the second parameter should be all lower case. Given that it's not just the movie title that's passed in, `movie` is probably a better name for it.